sys/test_utils/print_stack_usage/print_stack_usage.c

```diff
@@ -20,11 +20,37 @@
 #include "thread.h"
 #include "log.h"
 
+#if MODULE_FMT
+#include "fmt.h"
+#endif
+
+#if MODULE_FMT
+/* fmt's `print_str()` needs very little stack. ~200 total was fine on Cortex-M. */
+# define MIN_SIZE   (256)
```

Comment:
```suggestion
# define MIN_SIZE   THREAD_STACKSIZE_TINY
```

That happens to be 256 on Cortex-M. It is only smaller on AVR, where it would be 128. I think being able to use fmt should be a reasonable lower bound for `THREAD_STACKSIZE_TINY` - otherwise one IMO really should go for `THREAD_STACKSIZE_IDLE` if the thread is so trivial that it does no function calling or so.